storage/innobase/buf/buf0dblwr.cc

```diff
@@ -465,6 +474,8 @@ void buf_dblwr_t::recover()
   }
 
   recv_sys.dblwr.pages.clear();
+  for (auto it : encrypted_pages)
+    recv_sys.dblwr.pages.push_back(it);
```

Comment:
Usually the variable `it` refers to `iterator`, while here it is referring to a data item. I would suggest to write
```c++
  for (byte *page : encrypted_pages)
    recv_sys.dblwr.pages.push_back(page);
```